### ISSUE TITLE
.NET Core minification not working for form HttpPost

### ISSUE THREAD

**Issue by Xeevis** [Taritsyn/WebMarkupMin#34]
I've followed instructions to implement AspNetCore1 2.3.0 into .NET Core web app. Minification works fine except for single scenario.

I've made a simple form that posts to controller, after the post page source isn't minified.

```csharp
public class SampleController : Controller
{
    [HttpGet]
    public IActionResult Verify()
    {
        return View(); // Minified
    }

    [HttpPost]
    public IActionResult Verify(FoundersVerifyViewModel viewModel)
    {
        ModelState.AddModelError(string.Empty, "Form posted.");
        return View(); // No minification
    }
}
```
And the view
```html
@Html.ValidationSummary()
<form asp-controller="Sample" asp-action="Verify" method="post" class="form-horizontal">
    <button type="submit" class="btn btn-default">Verify</button>
</form>
```
You can't notice this with "View Page Source" as it makes new request to the page, but it's apparent if your page displays differently minified/unminified or if you have comments in the html which after form post will appear in the developer tools.

**Comment by Taritsyn** [comment]
Hello, Liam!

This was done on purpose, because there is no point to minify the response, which can not be cached.

**Comment by Xeevis** [comment]
I see, would be nice to have override that would minify all responses though. I'm not limited by processing power, but I do have limits on bandwidth, but I'd enable it even if for just consistency sake, added millisecond or two aren't such a big deal even if it's not cached. Isn't there bool or attribute I can add to force it?

**Comment by Taritsyn** [comment]
> Isn't there bool or attribute I can add to force it?

In next release I will implement this feature. Most likely, it will be the property on the principle of action is similar to `SupportedMediaTypes` property.

**Comment by aras777** [comment]
Hi, I have the same issue. In my case I am using cache in post requests using appcache from html5. So, I am happy that not only me has this issue, I will be checking for new version. In other cases minifier works great - keep up the good work.

**Comment by aras777** [comment]
Anyway I was able to fix it by adding line inside post action "Request.Method = "GET";", dirty workaround but works till next release :)

**Comment by Taritsyn** [comment]
@aras777  @Xeevis This problem is solved in [version 2.4.0](https://github.com/Taritsyn/WebMarkupMin/releases/tag/v2.4.0).

Use the following settings:

```csharp
services.AddWebMarkupMin(options =>
	{
		…
	})
	.AddHtmlMinification(options =>
	{
		options.SupportedHttpMethods = new HashSet<string> { "GET", "POST" };
		…
	})
	…
	;
```
